### PR TITLE
Update domain-check-2.sh

### PR DESCRIPTION
**Pull request by copenhaus**
-   modified .jp domain for its variations
-   Added support for .kr/.hk/.pt/.sg domains

### CODE REVIEW

**Comment by copenhaus** [comment]
- modified .jp domain for its variations
- added support for .kr/.hk/.pt/.sg domains

**Comment by click0** [comment]
Can you give examples of domains for these domain zones?

**Comment by copenhaus** [comment]
google.co.jp
google.jp

BUT it does look like my "substr" on line 499 needs some work.

**Comment by click0** [comment]
> 
> 
> google.co.jp
> google.jp
> 
> BUT it does look like my "substr" on line 499 needs some work.

```
# sh domain-check-2.sh -V
Script version:  2.52

# sh domain-check-2.sh -d google.jp

Domain                              Registrar                                      Status   Expires     Days Left
----------------------------------- ---------------------------------------------- -------- ----------- ---------
google.jp                           Google LLC                                     Valid    31-may-2021   99
```

I will only add changes adding support for .co.jp/.kr/.hk/.pt/.sg domains. 
I will ignore all other changes to "improvements" in the script syntax.
Also, we still need sample domains in .kr/.hk/.pt/.sg zones, in addition to the google domain.

File: domain-check-2.sh [hunk 1]
```diff
@@ -695,8 +760,7 @@ check_domain_status()
         "${TLDTYPE}" == "se" -o "${TLDTYPE}" == "nu" -o "${TLDTYPE}" == "dk" -o "${TLDTYPE}" == "it" -o \
         "${TLDTYPE}" == "do" -o "${TLDTYPE}" == "ro" -o "${TLDTYPE}" == "game" ];
     then
-        tdomdate=`${AWK} '/Registrar\ Registration\ Expiration\ [Dd]ate:|Registry\ Expiry\ Date:|Expiration\ [Dd]ate:|Renewal\ date:|Expir[ey]\ [Dd]ate:|Expires\ On:|[Ee]xpires:/ \
-           { print $NF; }' ${WHOIS_TMP} | ${AWK} -FT '{ print $1}' | ${HEAD} -1`
+        tdomdate=`${AWK} '/Registrar\ Registration\ Expiration\ [Dd]ate:|Registry\ Expiry\ Date:|Expiration\ [Dd]ate:|Renewal\ date:|Expir[ey]\ [Dd]ate:|Expires\ On:|[Ee]xpires:/ { print $NF; }' ${WHOIS_TMP} | ${AWK} -FT '{ print $1}' | ${HEAD} -1`
```
Comment (click0) [inline domain-check-2.sh:763]: We need to try to write code in such a way as to fit within a line length of 80 characters

File: domain-check-2.sh [hunk 2]
```diff
@@ -1,15 +1,20 @@
 #!/usr/bin/env bash
-
+#
```
Comment (click0) [inline domain-check-2.sh:2]: These changes must be left in your version of the script.

File: domain-check-2.sh [hunk 3]
```diff
@@ -613,13 +633,58 @@ check_domain_status()
 
     elif [ "${TLDTYPE}" == "jp" ]; # for .jp fixed @click0 2019/06/26
     then
-        tdomdate=`${AWK} -F] '/\[有効期限\]|\[Expires on\]/ {print $2}' ${WHOIS_TMP} | ${TR} -d " \r"`
+        tdomdate=`${AWK} -F] '/\[有効期限\]|\[Expires on\]|\[State\]|\[Connected \]/ {print $2}' ${WHOIS_TMP} | ${TR} -d " \r"`
+        tdomdate=$(echo $tdomdate| ${CUT} -d'(' -f2|${CUT} -d')' -f1)
```
Comment (click0) [inline domain-check-2.sh:637]: These two lines can be combined.

**Comment by click0** [comment]
Added support for .kr/.hk/.pt/.sg domains -- @copenhaus 23d7be38301bb25023fde0d74a358e20d7bc5b7c

**Pull request closed without merging**